plugins/amoeba/serialization/src/AmoebaVdwForceProxy.cpp

```diff
@@ -85,7 +85,7 @@ void AmoebaVdwForceProxy::serialize(const void* object, SerializationNode& node)
             type.setDoubleProperty("sigma", sigma).setDoubleProperty("epsilon", epsilon);
         }
         SerializationNode& pairs = node.createChildNode("TypePairs");
-        for (int i = 0; i < force.getNumParticleTypes(); i++) {
+        for (int i = 0; i < force.getNumTypePairs(); i++) {
```

Comment:
Is this a bugfix?

Reply:
Correct.

Reply:
Great! Let's just document this in the top of the PR so we can list this bugfix when we write release notes?

Reply:
It fixed a bug that was just introduced a few weeks ago in #2765.  The release notes only list fixes to bugs that were present in previous releases, not ones that were created and then fixed in the course of a single release.